__tests__/Unit/Components/Tasks/DependencyList.test.tsx

```diff
@@ -54,6 +42,11 @@ describe('DependencyList', () => {
         );
         const loading = screen.getByText(/Loading.../i);
         expect(loading).toBeInTheDocument();
+        expect(screen.getByText(/Loading/i)).toBeInTheDocument();
+
+        const noDependency = screen.queryByText('No Dependencies');
+
+        expect(noDependency).toBeNull();
```

Comment:
If you're testing for "No dependencies" then how are you checking that it should be null?
Shouldn't you check the existence of it in the document and not.toBeNull()?

Reply:
I have empty dependency so it should be null

Reply:
No dependency appear when dependsOn array is empty so i have passed empty array

Reply:
That's fine. but what of 
```
const noDependency = screen.queryByText('No Dependencies');
expect(noDependency).toBeNull();
```

You have queried text element with "No dependencies" and then expected it to be null
Shouldn't your screen contain it?

Reply:
done